lib/game_stats.rb

```diff
@@ -123,15 +116,13 @@ def average_goals_by_season
   end
 
   def average_game_goals
-    average = total_scores.sum / @games.size.to_f
-    average.round(2)
+    (total_scores.sum / @games.size.to_f).round(2)
   end
 
-  def wins_percentage(team)
+  def wins_percentage(site)
```

Comment:
What is site?

Reply:
This one was tricky to name for sure, I googled what the status of home vs away is called and found no common name, only suggestions for site, berth, status, etc. Host implies home in my opinion. Team_status crossed my mind too, but felt close to team_stats.